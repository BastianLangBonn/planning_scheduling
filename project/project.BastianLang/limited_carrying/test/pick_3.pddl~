(define (problem problem1)
	(:domain domain)
	(:objects 	s - s1
			o1 o2 o3 - o1)
	(:init 	(robot_at s)
		(at_location o1 s)
		(at_location o2 s)
		(at_location o3 s)
		(gripper_free)	
		(carries_none)	

		;Distance Metric

		(= (distance s s) 0)
		
		(= (total-cost) 0)
		
	)
	(:goal
		(carries_three)
	)
)
