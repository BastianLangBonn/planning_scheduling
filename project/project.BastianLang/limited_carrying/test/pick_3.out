(pick_3 o s)
